### PR TITLE
updated README.md file and added contribution guideline

### LINKED ISSUE #4: [DOCS] Add CONTRIBUTING GUIDELINES
Add contributing guidelines.

### PR DESCRIPTION
**Pull request by fk00750**
fixes #4

[DOCS] Add CONTRIBUTING GUIDELINES #4 

Summary
- Added the seperate file of CONTRIBUTION.md .Why?
   -  To keep the README md file clean and maintainable

- CONTRIBUTION.md file is divided into following sub sections.
     - Contributor License Agreement.
     - Contributor Code of Conduct.
         - inked to Code of Conduct of Collab Community.
    -  Contribution Guideline includes.
          - Guide for first type contributors
          - Guide for adding problems solutions

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### If you are adding a solution
+
+- If you are providing a solution, please ensure that it is your own.
+- Double Check your Solution.
+- Provide a test case for your solution
+- Insert the solution into [solution folder](Solution) of your prefered language ([C++](Solution/cpp) or [Java](Solution/java) )
+
+```
+📦Leetcode-problem
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:25]: Please delete:

```
📦Leetcode-problem
|___.github
|___Problem
|___Solution
|   |   cpp
|   |   java
|___CONTRIBUTING.md
|___README.md
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (KendallDoesCoding) [reply CONTRIBUTING.md:25]: @fk00750 Please do this at the earliest

Comment (fk00750) [reply CONTRIBUTING.md:25]: @KendallDoesCoding  Ok

**Comment by fk00750** [comment]
hey @KendallDoesCoding can you review these changes?

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Collab-Community
+Copyright (c) 2022 collab-community/Leetcode-problem
```
Comment (KendallDoesCoding) [inline LICENSE:3]:
```suggestion
Copyright (c) 2022 collab-community/Leetcode-problem
```

The copyright is the name of the person, company, organization who owns the copyright, we don't need to inculde the project name haha

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Rest looks good but please revert this!

File: LICENSE (hunk 2 above)
Comment (fk00750) [reply LICENSE:3]: ok got it, I thought the copyright should be on the name of the project haha

Comment (KendallDoesCoding) [reply LICENSE:3]: That works too, but not required!

Comment (KendallDoesCoding) [reply LICENSE:3]: and not preferred as well.

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @fk00750!

**Comment by fk00750** [comment]
Also thanks to you @KendallDoesCoding and all of the team for providing me with such a great opportunity.

**Comment by KendallDoesCoding** [comment]
> Also thanks to you @KendallDoesCoding and all of the team for providing me with such a great opportunity.

ofc! we're here to help you and to guide coders..

**Comment by KendallDoesCoding** [comment]
even we learn somethings from yall :)

**Pull request merged**